api/turing/models/ensembling_job.go

```diff
@@ -56,8 +56,8 @@ func (r *JobConfig) Scan(value interface{}) error {
 type InfraConfig struct {
 	ArtifactURI        string                       `json:"artifact_uri"`
 	EnsemblerName      string                       `json:"ensembler_name"`
-	ServiceAccountName string                       `json:"service_account_name"`
-	Resources          *BatchEnsemblingJobResources `json:"resources"`
+	ServiceAccountName string                       `json:"service_account_name" validate:"required"`
+	Resources          *BatchEnsemblingJobResources `json:"resources" validate:"required"`
```

Comment:
Why is this required but the fields within the `BatchEnsemblingJobResources` are all optional?

Reply:
This is a mistake, will remove this